rpc_jobs/standard_job_premerge.yml

```diff
@@ -138,13 +139,14 @@
                   }} // withCredentials
                 }} // dir
               }} // ansiColor
-            }} catch (e) {{
-              print(e)
-              currentBuild.result="FAILURE"
-              throw e
             }} finally {{
+              dir(env.RE_HOOK_RESULT_DIR){{
+                junit allowEmptyResults: true, testResults: '*.xml'
+              }}
+              stage("Archive Artifacts"){{
```

Comment:
The ``archive_artifacts`` function already specifies a stage - no need for nesting it in another one.

Reply:
Good spot, done.